internal/extraccion_test.go

```diff
@@ -0,0 +1,102 @@
+package internal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/html"
+)
+
+func TestCargarDocumento(t *testing.T) {
+	filePath := "../data/menu.html"
+
+	doc, err := cargarDocumento(filePath)
```

Comment:
os expliqué lo de Arrange, Act and Assert. ¿Crees de verdad que tienes que testear la carga de un fichero? ¿o sería simplemente la parte "Arrange"?
Se testean comportamientos, no funcionalidades.

Reply:
Resuelto en [7a681ba](https://github.com/antoniorr02/MenuConsulter/pull/22/commits/7a681ba81ca138f6703c8641abcf495de2ec0310)